build-ci/jenkins-stack-os-matrix.rst

```diff
@@ -0,0 +1,60 @@
+##############################################################
+Testing the LSST DM Stack with the Jenkins stack-os-matrix Job
+##############################################################
+
+`stack-os-matrix`_ is the continuous integration (CI) job that DM developers must run to ensure tests pass before merging changes to any LSST EUPS Stack package.
+This page explains how to run the `stack-os-matrix`_ job and find build and test logs.
+
+For more information about the DM's Jenkins continuous integration service, see :doc:`jenkins`.
+
+Running a stack-os-matrix job
+=============================
+
+1. Open the `stack-os-matrix dashboard <stack-os-matrix>`_ in a browser directly or by clicking on the **stack-os-matrix** job from the main dashboard at https://ci.lsst.codes.
+
+2. Click on the **Run** button near the top of the page.
+
+3. Configure the run in the pop-up dialog:
+
+   - **Set the list of Git refs** (branch or tag names) to check out from Stack Git repositories.
+
+     For example, if you enter ``tickets/DM-2 tickets/DM-1``, the build system will attempt to check out the ``tickets/DM-2`` branch in Stack packages.
+     If a package does not have a ``tickets/DM-2`` branch, it will attempt to check out the ``tickets/DM-1`` branch.
+     If a package has neither branch, it falls back to checking out the ``master`` branch.
+
+   - **Set the list of EUPS packages** to build.
+     Use the default (``lsst_distrib``) to build and test your changes with a full Stack.
+     To improve build times you can instead specify the name of the package you are actively developing (see also **Skip Demo**, below).
```

Comment:
Was Skip Demo meant to be a link?  I don't think it has to be, but usually when you say "See something" the something is a link.